src/pages/index.js

```diff
@@ -1,5 +1,4 @@
-import React,{ useState, useEffect } from 'react';
-import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
+import React from 'react';
```

Comment:
Is it needed?

Reply:
Yes it is, when i remove it i get this error ReferenceError: React is not defined